tests/functional/object/mcg/test_custom_credentials_using_mcg_cli.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import string
+import random
+
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    MCGTest,
+    tier1,
+    red_squad,
+    mcg,
+)
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import retrieve_cli_binary
+from ocs_ci.utility.utils import run_cmd, get_random_str
+from ocs_ci.helpers.helpers import get_s3_credentials_from_secret
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+mcg_cli = constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH
+
+
+@tier1
+@red_squad
+@mcg
+class TestCustomCredsUsingMcgCli(MCGTest):
+    def create_nb_account(self, account_name):
+        "Create noobaa account"
+        logger.info("Creating account using noobaa-CLI")
+        output = run_cmd(
+            cmd=f"{mcg_cli} account create {account_name} -n openshift-storage"
+        )
+        logger.info(output)
+
+    def delete_nb_account(self, account_name):
+        "Delete noobaa account"
+        logger.info("deleting account")
+        output = run_cmd(
+            cmd=f"{mcg_cli} account delete {account_name} -n openshift-storage"
+        )
+        logger.info(output)
+
+    def update_nb_account(self, account_name, access_key, secret_key):
+        "Update noobaa account"
+        output = run_cmd(
+            cmd=f"{mcg_cli} account credentials {account_name} "
+            + f"--access-key={access_key} "
+            + f"--secret-key={secret_key} "
+            + "-n openshift-storage",
+            ignore_error=True,
+        )
+        logger.info(output)
+
+    @skipif_ocs_version("<4.17")
+    def test_account_update_with_custom_creds_using_CLI(self):
```

Comment:
For function naming consistency, please use 'cli' in lowercase instead.

Reply:
Fixed